Tests/StoreKitUnitTests/OfflineEntitlements/PurchasedProductsFetcherTests.swift

```diff
@@ -18,20 +18,32 @@ import XCTest
 @testable import RevenueCat
 
 @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
-class PurchasedProductsFetcherTests: StoreKitConfigTestCase {
+class BasePurchasedProductsFetcherTests: StoreKitConfigTestCase {
 
-    private var sandboxDetector: SandboxEnvironmentDetector!
-    private var fetcher: PurchasedProductsFetcher!
+    fileprivate var sandboxDetector: SandboxEnvironmentDetector!
+    fileprivate var fetcher: PurchasedProductsFetcher!
 
     override func setUpWithError() throws {
         try super.setUpWithError()
 
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
 
         self.sandboxDetector = MockSandboxEnvironmentDetector(isSandbox: .random())
-        self.fetcher = PurchasedProductsFetcher(sandboxDetector: self.sandboxDetector)
+        self.fetcher = PurchasedProductsFetcher(
+            appStoreSync: self.syncAppStore,
+            sandboxDetector: self.sandboxDetector
+        )
     }
 
+    fileprivate func syncAppStore() async throws {
```

Comment:
Could this be `appStoreSync`, same as in the other places?

Reply:
My thinking is that the closure is a noun "App Store sync closure", versus this is an action / verb: "sync App Store".
Let me know if that's confusing though.